### PR TITLE
Review

### PR DESCRIPTION
**Pull request by machine-del**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    <button class="actions__button notifications">
+                        <img src="/images/icons/icon-header-notifications.png" alt="#">
+                    </button>
+                </div>
+            </div>
+        </div>
+    </header>
+    
+    <main>
+        <section class="control-panel">
```
Comment (Adilxa) [inline index.html]: aside

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                                <li><span class="active-img"><img src="/images/icons/icon-left-panel-home.png" alt="#"></span><a href="#" class="active">Панель управления</a></li>
+                                <li><span><img src="/images/icons/icon-left-panel-chart.png" alt="#"></span><a href="#">Таблицы</a></li>
+                                <li><span><img src="/images/icons/icon-left-panel-card.png" alt="#"></span><a href="#">Выставление счетов</a></li>
+                                <li><span><img src="/images/icons/icon-left-panel-build.png" alt="#"></span><a href="#">RTL</a></li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <div class="control-panel__item">
+                        <div class="control-panel--title">
+                            Страницы аккаунта
```
Comment (Adilxa) [inline index.html]: do h6 , h5

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                            <ul>
+                                <li><span><img src="/images/icons/icon-left-panel-user.png" alt="#"></span><a href="#">Профиль</a></li>
+                                <li><span><img src="/images/icons/icon-left-panel-document.png" alt="#"></span><a href="#">Войти</a></li>
+                                <li><span><img src="/images/icons/icon-left-panel-rocket.png" alt="#"></span><a href="#">Зарегистрироваться</a></li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <div class="control-panel__item">
+                        <img class="documentation" src="images/icons/icon-left-panel-documentation.png" alt="#">
+                        <div class="control-panel__subtitle">Нужна помощь?</div>
```
Comment (Adilxa) [inline index.html]: a tag

**Review by Adilxa — COMMENTED** [review]
Not good to me (

**Comment by machine-del** [comment]
Corrected

**Pull request merged**